controllers/surveyController.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../models/Survey");
+
+module.exports = {
+  findAllSurveys: function (req, res) {
+    db.find({})
+      .then(function (dbArticle) {
+        res.json(dbArticle);
+      })
+      .catch(function (err) {
+        res.json(err);
+      });
+  },
+
+  findSurveyById: function (id, res) {
+    db.findById(id)
+      .then(function (survey) {
+        res.json(survey);
```

Comment:
Could we use return here instead so the survey can be manipulated into an html page on the backend in the get routes and then send that to the front end

Reply:
OK, I'll do that right now